ThemeDecorator/ThemeDecorator.js

```diff
@@ -280,10 +287,10 @@ const ThemeDecorator = hoc(defaultConfig, (config, Wrapped) => {
 			);
 
 			return (
-				<React.Fragment>
+				<ThemeContext.Provider value={{accent, highlight}}>
```

Comment:
accent, highlight should be skin's default color, not fixed color if app developer doesn't set

Reply:
Depending on the current skin, I set the default accent and highlight color.

For other components, those color are defined in less files. But `ArcPicker` and `ArcSlider` uses SVG format, then can't use them in the less files. So we have to define them in JavaScript files - ThemeDecorator.js separately. That's the reason why I have to define the `defaultColors` variable in it.